doc/workflows_and_tools/enrollment_tools/yubikey_enrollment_tools.rst

```diff
@@ -0,0 +1,136 @@
+.. _yubikey_enrollment_tools:
+
+Yubikey Enrollment Tools
+------------------------
+
+.. index:: Yubikey, Yubico AES mode, Yubikey OATH-HOTP mode
+
+The Yubikey can be used with privacyIDEA in Yubico's own AES mode (*Yubico OTP*),
+in the HOTP mode (*OATH-HOTP*) or the seldomly used static password mode.
+
+This section describes several tools which can be used to initialize and enroll a
+Yubikey with privacyIDEA.
+
+If not using the :ref:`yubico_token` mode, the Yubikey has to be initialized with privacyIDEA
+to bring the token secrets to the system.
+
+privacyIDEA ships several tools to (mass-)enroll Yubikeys in AES mode (Yubikey Token) or HOTP mode (HOTP Token).
+
+.. _privacyideaadm_enrollment:
+
+.. index:: privacyideaadm, admin tool
+
+privacyidea CLI tool
+~~~~~~~~~~~~~~~~~~~~
+
+For Linux Clients, there is the ``privacyidea`` command line
+client [#privacyideaadm]_, to initialize the Yubikeys. You can use the mass enrollment, which
+eases the process of initializing a whole bunch of tokens.
+
+Run the command like this::
+
+   privacyidea -U https://your.privacyidea.server -a admin token \
+   yubikey_mass_enroll --yubimode YUBICO
+
+This command initializes the token and stores the AES secret and prefix
+in privacyIDEA, so the token is immediately useful. You can enroll Yubikeys
```

Comment:
"immediately useful" sounds awkward.

The token now exits in privacyIDEA. The admin still has to assign the token to a user.
So I would do s.th. like.

"This command initializes the token and stores the AES secret and prefix
in privacyIDEA. The administrator can now assign the token to a user."